### PR TITLE
e2e expand collapse large values

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/helpers/api/api-keys.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    HashKeyParameters,
+    ListKeyParameters,
+    SetKeyParameters,
+    SortedSetKeyParameters,
+    StreamKeyParameters
+} from '../../pageObjects/browser-page';
+import { getDatabaseByName } from './api-database';
+
+const endpoint = apiUrl;
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'; // lgtm[js/disabling-certificate-validation]
```
Comment (ofersteinberg) [inline tests/e2e/helpers/api/api-keys.ts:15]: Don't you already use it in past? Maybe you come to move it to a common place and import it?

File: tests/e2e/tests/regression/browser/large-key-details-values.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        await addListKeyApi(listKeyParameters, ossStandaloneConfig);
+    })
+    .afterEach(async() => {
+        //Delete database
+        await deleteKeysApi(keyNames, ossStandaloneConfig);
+        await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    });
+test('Verify that user can click on a row to expand it if any of its cells contains a value which is truncated.', async t => {
+    const entryFieldLong = browserPage.streamEntryFields.nth(1).parent(1);
+    const entryFieldSmall = browserPage.streamEntryFields.nth(0).parent(1);
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/browser/large-key-details-values.e2e.ts:47]: Can't you use data-id?

Comment (vlad-dargel) [reply tests/e2e/tests/regression/browser/large-key-details-values.e2e.ts:47]: this is a component common to all virtualized table cells, and for all other tables, it is better not to add data-test-id
this component does not know the context of where it is being drawn

File: tests/e2e/helpers/api/api-keys.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/helpers/api/api-keys.ts]: moved to conf.ts
thanks!

File: tests/e2e/tests/regression/browser/large-key-details-values.e2e.ts (hunk 2 above)
Comment (ofersteinberg) [reply tests/e2e/tests/regression/browser/large-key-details-values.e2e.ts:47]: Notice it can be break

File: tests/e2e/helpers/api/api-keys.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+/**
+ * Add Stream key
+ * @param keyParameters The key parameters
+ * @param databaseParameters The database parameters
+ */
+export async function addStreamKeyApi(keyParameters: StreamKeyParameters, databaseParameters: AddNewDatabaseParameters): Promise<void> {
+    const databaseId = await getDatabaseByName(databaseParameters.databaseName);
+    const response = await request(endpoint).post(`/instance/${databaseId}/streams`)
```
Comment (ofersteinberg) [inline tests/e2e/helpers/api/api-keys.ts:41]: You can create one generic function for all the below functions.
See that `/instance/${databaseId}/` is common, and the last part is changed.
WDYT?

**Review by ofersteinberg — APPROVED** [review]

File: tests/e2e/helpers/api/api-keys.ts (hunk 3 above)
Comment (vlad-dargel) [reply tests/e2e/helpers/api/api-keys.ts:41]: they are related to different types, so better to leave as it is

**Pull request merged**